project/plugins.sbt

```diff
@@ -1,3 +1,8 @@
+libraryDependencies += Defaults.sbtPluginExtra(
+  "com.eed3si9n" % "sbt-assembly" % "0.14.10",
+  (sbtBinaryVersion in pluginCrossBuild).value,
+  (scalaBinaryVersion in pluginCrossBuild).value)
```

Comment:
Why all this vs `addSbtPlugin` ?

Reply:
Overriding the version used in `sbt-multi-jvm`

Reply:
Ah, so it's updating a transitive dependency. Thanks for explaining!

Reply:
Now I just have to beat @scala-steward to the eventual plugin version upgrade and will remove the override in that PR.